src/transformers/models/xlm_roberta/tokenization_xlm_roberta.py

```diff
@@ -128,10 +139,17 @@ def __init__(
             cls_token=cls_token,
             pad_token=pad_token,
             mask_token=mask_token,
+            enable_sampling=enable_sampling,
+            nbest_size=nbest_size,
+            alpha=alpha,
```

Comment:
Talking about it with @sgugger, we think this might be better suited in some `**sp_model_kwargs` or something along those lines, that we could then add to all SentencePiece-based tokenizers. How does that sound?

Reply:
Ok - cool. I made the change.